lib/connection.c

```diff
@@ -1142,3 +1124,20 @@ void STDCALL sf_header_destroy(SF_HEADER *sf_header) {
     curl_slist_free_all(sf_header->header);
     SF_FREE(sf_header);
 }
+
+//Help function to log the useful response json data
+void STDCALL create_json_resp_log(cJSON **json){
```

Comment:
@sfc-gh-kwagner Could you help me take a look about the code standard since I am not familiar with it. Not sure if I need to add STDCALL or not. Thanks!

Reply:
You need STDCALL for when people compile this on windows